website/content/docs/plugins/index.mdx

```diff
@@ -33,8 +36,48 @@ binaries can be obtained from [releases.hashicorp.com](https://releases.hashicor
 or they can be [built from source](/docs/plugins/plugin-development#building-a-plugin-from-source).
 
 Vault's external plugins are completely separate, standalone applications that
-Vault executes and communicates with over RPC. Each time a Vault secret engine
-or auth method is mounted, a new process is spawned. However, plugins can be
-made to implement [plugin multiplexing](/docs/plugins/plugin-architecture#plugin-multiplexing)
+Vault executes and communicates with over RPC. Each time a Vault secret engine,
+auth method, or database plugin is mounted, a new process is spawned. However,
+plugins can be made to implement [plugin multiplexing](/docs/plugins/plugin-architecture#plugin-multiplexing)
 to improve performance. Plugin multiplexing allows plugin processes to be
 reused across all mounts of a given type.
+
+## Plugin Versioning
+
+Starting in 1.12.0, Vault supports managing, running and upgrading plugins using
```

Comment:
nit: same comment as above. I don't think we need the 1.12.0 version called out here since we have versioned docs.